lib-src/libnyquist/nyx.c

```diff
@@ -1060,9 +1044,6 @@ int nyx_get_audio(nyx_audio_callback callback, void *userdata)
       bool terminated = true;
       // how many samples to compute before calling callback:
       int64_t togo = max_sample_block_len;
-      if (nyx_input_length > 0 && total + togo > nyx_input_length) {
-         togo = nyx_input_length - total;
-      }
```

Comment:
"blame" shows these lines were added at cd79f1099b0ca59167dfd895e0d06f99123b44e8 to fix a bug involving Allegro.  That bug fix is also unimpaired by this deletion?

Reply:
I should have split cd79f10 -- there must have been an unrelated bug fix involving Allegro. I can't imagine any interaction with the addition or deletion of these lines.